src/sgx/CMakeLists.txt

```diff
@@ -10,92 +10,127 @@
 #        module_cache
 #        system
 #        )
+
+# NASM configuration
```

Comment:
Just to improve readability for others I've added some linebreaks and comments in here.